Modules/_functoolsmodule.c

```diff
@@ -422,20 +499,22 @@ partial_vectorcall(partialobject *pto, PyObject *const *args,
             }
         }
         assert(j == pto_phcount);
-        if (nargskw > pto_phcount) {
-            memcpy(stack + pto_nargs, args + j, (nargskw - j) * sizeof(PyObject*));
+        /* Add remaining args from new_args */
+        if (nargs > pto_phcount) {
+            memcpy(stack + pto_nargs, args + j, (nargs - j) * sizeof(PyObject*));
```

Comment:
When using `sizeof` on a structure, you cannot use pointer types, because pointer types are **8 bytes** on most machines, not the size of the structure.

Reply:
And which one do we need here?